openr/spark/tests/MockIoProvider.cpp

```diff
@@ -0,0 +1,345 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
Pls remove the change here for MockIoProvider.h and MockIoProvider.cpp from this patch.

We have migrated and move it under openr/tests/mocks/MockIoProvider.h + .cpp

The commit name is: [openr][system-service] (8/N): Split IoProvider from Spark lib for MOCK lib usage